nvflare/fuel/hci/server/file_transfer.py

```diff
@@ -311,6 +323,58 @@ def _unzip_data(self, job_data, job_id):
         if workspace_bytes is not None:
             unzip_all_from_bytes(workspace_bytes, workspace_dir)
 
+    def download_job_single_file(self, conn: Connection, args: List[str]):
+        if len(args) != 3:
+            conn.append_error("syntax error: job ID and the path to the file to download are required")
+            return
+
+        job_id = args[1]
+        file = args[2]
```

Comment:
suggest have something data_source = args[3]
data_source need to be either "workspace" or "job"

So use data_source to do if condition rather than file.startswith